### DISCUSSION TITLE
Nesting multi-layer routes

### DISCUSSION THREAD

**Discussion by zareef26** [remix-run/remix#6960] [Q&A] [answered]
I am trying to create pages for `/admin/subjects` and `/admin/subjects/new` routes using v2_routing. I have named my page files `admin.subjects.jsx` and `admin.subjects.new.jsx` for the above routes, respectively.
Now, `/admin/subjects` route is working fine, but hitting `/admin/subjects/new` route is showing the same page from `/admin/subjects` route.
Is there any special method for those kind of routing? If I want to nest my routes deeper, how could I do that?

**Comment by kiliman** [comment, marked as answer]
By default, in `v2` routing, child routes will automatically nest under a parent route with the same prefix. In your case, Remix is seeing *admin.subjects.tsx* as the **parent** layout for *admin.subjects.new.tsx*, since they both have the common prefix of `admin.subjects`.

The reason your new route was still showing `/admin/subjects` was that Remix was using it as a layout, and if it didn't have an `<Outlet>` it won't show the child route.

To let Remix know **not** to nest, add a trailing `_` in your route.

`admin.subjects_.new.tsx` <- this says "*admin.subjects* should **not** be a parent layout"
